src/main/java/org/candlepin/subscriptions/tally/billing/admin/InternalBillingController.java

```diff
@@ -111,6 +119,10 @@ private List<MonthlyRemittance> transformUsageToMonthlyRemittance(
     }
 
     for (RemittanceSummaryProjection entity : remittanceSummaryProjections) {
+      // Remove this null assignment once we start adding statuses in prod
+      // https://issues.redhat.com/browse/SWATCH-2289
+      var remittanceStatus =
+          Objects.nonNull(entity.getStatus()) ? entity.getStatus().getValue() : "null";
```

Comment:
Seems like this should be null and not string "null"
```suggestion
          Objects.nonNull(entity.getStatus()) ? entity.getStatus().getValue() : null;
```

Reply:
No we have to report status as null hence we are giving String null. We will remove this code once this is deployed in prod since it is mostly to support existing records in billable_usage_remittance.